Stripe/PublicHeaders/STPPaymentMethodBillingDetails.h

```diff
@@ -19,27 +20,27 @@ NS_ASSUME_NONNULL_BEGIN
  
  @see https://stripe.com/docs/api/payment_methods/object#payment_method_object-billing_details
  */
-@interface STPPaymentMethodBillingDetails : NSObject <STPAPIResponseDecodable>
+@interface STPPaymentMethodBillingDetails : NSObject <STPAPIResponseDecodable, STPFormEncodable>
 
 /**
  Billing address.
  */
-@property (nonatomic, nullable) STPPaymentMethodBillingDetailsAddress *address;
+@property (nonatomic, strong, nullable, readwrite) STPPaymentMethodBillingDetailsAddress *address;
```

Comment:
any particular reason for adding explicitly `strong` and `readwrite` (which are the defaults)?

Reply:
Haha ahh I added them after reading the Styleguide https://github.com/stripe/stripe-ios/blob/master/STYLEGUIDE.md#properties

I interpreted it as saying we should be explicit.